ignition/service/logging.py

```diff
@@ -17,9 +17,21 @@
 
 PRIVATE_KEY_PREFIX = '-----BEGIN RSA PRIVATE KEY-----'
 PRIVATE_KEY_SUFFIX = '-----END RSA PRIVATE KEY-----'
-PRIVATE_KEY_REGEX = re.compile('{0}(.*?){1}'.format(PRIVATE_KEY_PREFIX, PRIVATE_KEY_SUFFIX), flags=re.DOTALL)
 OBFUSCATED_PRIVATE_KEY = '***obfuscated private key***'
 
+BEGIN_CERT_PREFIX = '-----BEGIN CERTIFICATE-----'
+END_CERT_SUFFIX = '-----END CERTIFICATE-----'
+OBFUSCATED_CERTIFICATE = '***obfuscated certificate***'
+
+TOKEN_PREFIX = 'token'
+TOKEN_SUFFIX = "'"
+
+PWD_PREFIX = "Password"
+PWD_SUFFIX = ","
+PWD_SUFFIX_WITH_NEXTLINE = '\\\\n '
```

Comment:
Why are there two suffixes here? Again are we assuming a password field is always terminated with a `'`?